src/mx/uaemex/fi/paradigmas_I/figuras/vista/PanelCirculoData.java

```diff
@@ -0,0 +1,45 @@
+package mx.uaemex.fi.paradigmas_I.figuras.vista;
+
+import mx.uaemex.fi.paradigmas_I.figuras.data.Figura;
+
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+
+public class PanelCirculoData extends PanelDeFigura{
+
+    private JSpinner radio;
+
+    public PanelCirculoData(){
+        super();
+
+        JLabel etqRadio;
+        SpinnerModel modelo;
+        JButton btnGD;
+
+        setLayout(new GridLayout(3,1));
+
+        modelo = new SpinnerNumberModel(0,0,1000,1);
+
+        etqRadio = new JLabel("Radio: ");
+        radio= new JSpinner();
+        radio.setModel(modelo);
+        btnGD = new JButton("Guarda datos");
+
+        add(etqRadio);
+        add(radio);
+        add(btnGD);
+
+        setBorder(new EmptyBorder(10,10,10,10));
+
+
+
+    }
+
+    @java.lang.Override
```

Comment:
Esta es una anotación, a pesar de ser probablemente correcta es mejor escribirla de la siguiente forma:
```suggestion
    @Override
```